tools/packaging/static-build/virtiofsd/build-static-virtiofsd.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ARCH=$(uname -m)
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+source "${script_dir}/../../scripts/lib.sh"
+
+config_dir="${script_dir}/../../scripts/"
+
+virtiofsd_repo="${virtiofsd_repo:-}"
+virtiofsd_dir="virtiofsd"
+virtiofsd_version="${virtiofsd_version:-}"
+kata_version="${kata_version:-}"
+
+if [ -z "$virtiofsd_repo" ]; then
+	info "Get virtiofsd information from runtime versions.yaml"
+        virtiofsd_url=$(get_from_kata_deps "externals.virtiofsd.url" "${kata_version}")
+	[ -n "$virtiofsd_url" ] || die "failed to get virtiofsd url"
+        virtiofsd_repo="${virtiofsd_url}.git"
+fi
+[ -n "$virtiofsd_repo" ] || die "failed to get virtiofsd repo"
+
+[ -n "$virtiofsd_version" ] || virtiofsd_version=$(get_from_kata_deps "externals.virtiofsd.version" "${kata_version}")
+[ -n "$virtiofsd_version" ] || die "failed to get virtiofsd version"
+
+if [ "${ARCH}" != "x86_64" ]; then
+	info "Only x86_64 binaries are distributed as part of the virtiofsd releases" && exit 1
+fi
+
+pull_virtiofsd_released_binary() {
+    info "Download virtiofsd version: ${virtiofsd_version}"
+    virtiofsd_zip=$(get_from_kata_deps "externals.virtiofsd.meta.binary" "${kata_version}")
+
+    mkdir -p virtiofsd
+
+    pushd virtiofsd
+    curl --fail -L ${virtiofsd_zip} -o virtiofsd.zip || return 1
+    unzip virtiofsd.zip
+    mv -f target/x86_64-unknown-linux-musl/release/virtiofsd virtiofsd
```

Comment:
Seeing as this doesn't seem to execute in a container (like other builds in `static-build` do),

`rm -rf target`

to clean up might be a good idea?

Reply:
It actually is executed inside a container, at least as part of our GitHub actions.

https://github.com/kata-containers/kata-containers/pull/4233/files#diff-08d854f59d53cefd732ccc081adc1b49e6279e0e9bb33218ac99bbd56589e922 even installed unzip in the dockerfile used, otherwise our CIs would fail.

Reply:
Updated!